docs/data-dictionary/types/soort_lening.yml

```diff
@@ -0,0 +1,8 @@
+type: string
+description: Er bestaan verschillende soorten leningen die verstrekt kunnen worden. Via dit veld wordt vastgelegd voor wat voor soort lening de borgstelling wordt aangevraagd.
+enum:
+  - DK
+  - PL
+  - SK
+  - SL
+example: SK
```

Comment:
Een terzijde: Ik denk dat DK een aparte gegevens verzameling is omdat hiervoor andere velden worden gebruikt dan bij de overige soorten leningen. Hierdoor zal het DK als soort lening obv ons ontwerp niet voor kunnen komen denk ik.

Reply:
Dan haak ik het weg.